example-config/uwsgi.service

```diff
@@ -0,0 +1,16 @@
+# systemd unit for running uwsgi in emperor mode
+# typical location of this file would be /etc/systemd/system/uwsgi.service
+
+[Unit]
+Description=uWSGI Emperor service
+
+[Service]
+ExecStartPre=/bin/bash -c 'mkdir -p /run/uwsgi; chown EtesyncUser:www-data /run/uwsgi'
```

Comment:
I'd rather have this line in the instructions rather than executed every time, don't you think?

Reply:
Isn't `/run` usually on a RAM based `tempfs` and so not preserved between boots?

Reply:
My bad, you're right.
Though actually, why make it a dir in the first place? Just put the socket directly as `/run/etesync_server.sock`. No? Saves us from having to do this special setup here.